go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/Azure/ARO-RP
 go 1.18
 
 require (
+	github.com/Azure/ARO-RP/pkg/api v0.0.0-00010101000000-000000000000
```

Comment:
Why such a strange suffix or is this just some Go convention?

Reply:
That's a Go convention :(

Reply:
Sometimes Go makes me sad